wongnung/models.py

```diff
@@ -0,0 +1,136 @@
+from typing import List
+
+from django.db import models
+from datetime import datetime
+import tmdbsimple as tmdb
+
+
+class Film(models.Model):
+    """
+    Model for Film with film ID
+
+    :param filmId: The ID of a specific film.
+    :type filmId: int
+    :param title: The title of a film.
+    :type title: str
+    :param year_released: The release year of the film.
+    :type year_released: int
+    :param director: The name of the director(s) of the film.
+    :type director: str
+    :param genres: The genres of the film.
+    :type genres: str
+    :param summary: A summary of the film.
+    :type summary: str
+    :param stars: The stars(actors and actresses) of the film
+    :type stars: str
+    """
+    filmId = models.IntegerField(primary_key=True)  # Required
+    title = models.CharField(max_length=256)  # Required
+    year_released = models.IntegerField()  # Required
+    director = models.CharField(max_length=512, blank=True)
+    genres = models.CharField(max_length=512, blank=True)
+    summary = models.CharField(max_length=1000, blank=True)
+    stars = models.CharField(max_length=1000, blank=True)  # names of
+
+    def get_director(self):
+        """Get a list contain the name of the director(s) of the film
+
+        :return: A list containing the string representing the name of the director(s) of the film
+        :rtype: list
+        """
```

Comment:
Change `rtype` in docstrings to type hinting, this is more supported for type checkers like Pyright, Pylint etc.
```suggestion
    def get_director(self) -> List[str]:
        """Get a list contain the name of the director(s) of the film

        :return: A list containing the string representing the name of the director(s) of the film
        """
```

Reply:
type hint added